content/howtos/setup-with-HA.md

```diff
@@ -9,7 +8,14 @@ Home Assistant is an open-source home automation platform that allows users to c
 
 Currently, Home Assistant supports conversation-based agents and services. As of writing this, OpenAIs API is supported as a conversation agent; however, access to your homes devices and entities is possible through custom components. Local based services, such as LocalAI, are also available as a drop-in replacement for OpenAI services.
 
-An example custom integration to utilize Local-based services in Home-LLM. Home-LLM is a Home Assistant integration developed by Alex O'Connell ([acon96](https://github.com/acon96)) that allows for a completely local Large Language Model acting as a personal assistant. Using LocalAI as the backend is one of the supported platforms .
+## There are multiple custom integrations available:
+
+Please note that both of the projects are similar in term of visual interfaces, they seem to be derived from the official Home Assistant plugin: [OpenAI Conversation](https://www.home-assistant.io/integrations/openai_conversation/) (to be confirmed)
+
+- Home-LLM is a Home Assistant integration developed by Alex O'Connell ([acon96](https://github.com/acon96)) that allows for a completely local Large Language Model acting as a personal assistant. Using LocalAI as the backend is one of the supported platforms. The provided Large Language Models are specifically trained for home assistant and are therefore smaller in size.
+- Extended OpenAI Conversation uses OpenAI API’s feature of function calling to call service of Home Assistant. Is more generic and work with most of the Large Language Model.
```

Comment:
unfortuantly, this is not entirely true. You can use an LLM for Extended OpenAI Conversation, but it will not utilize the fuctions properly and thus the servicies will not work.

Reply:
Not issue on my side (except to get the room temperature): Can get the status of a light and turn it off or on, I put the models in the documentation. I currently use the mixtral 13b one.